dart/test.dart

```diff
@@ -0,0 +1,9 @@
+import 'wilayat.dart';
```

Comment:
I suggest renaming the file to `algeria_cities.dart`

```suggestion
import 'algeria_cites.dart';
```